roles/matrix-bridge-hookshot/tasks/setup_uninstall.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Check existence of matrix-hookshot service
+  stat:
+    path: "{{ matrix_systemd_path }}/matrix-hookshot.service"
+  register: matrix_hookshot_service_stat
+
+- name: Ensure matrix-hookshot is stopped
+  service:
+    name: matrix-hookshot
+    state: stopped
+    enabled: no
+    daemon_reload: yes
+  when: "matrix_hookshot_service_stat.stat.exists"
+
+- name: Ensure matrix-hookshot.service doesn't exist
+  file:
+    path: "{{ matrix_systemd_path }}/matrix-hookshot.service"
+    state: absent
+  when: "matrix_hookshot_service_stat.stat.exists"
+
+- name: Ensure systemd reloaded after matrix-hookshot.service removal
+  service:
+    daemon_reload: yes
+  when: "matrix_hookshot_service_stat.stat.exists"
+
+# remove base_path?
```

Comment:
A note about removing the base path: you currently store some passkey underneath it.

If you are to remove it and then start fresh (config-wise and passkey-wise), but still use the old database (because nothing really drops it), what's going to happen?

Reply:
I have noticed that other roles don't seem to clean these kinds of things up too much, which has the benefit that you can enable it again later and it will continue where it left off. I left this in as a comment, but I think I'll stick to the usual behaviour, given that only a handful config files get stored in this base path.